modin/experimental/pandas/io_exp.py

```diff
@@ -154,16 +179,21 @@ def parser_func(
     return parser_func
 
 
-def _read(**kwargs):
+def _read(**kwargs) -> DataFrame:
     """
-    Read csv file from local disk.
+    General documentation in `modin.pandas.read_csv`.
+
+    Experimental feature is simultaneous reading from multiple csv files which are
+    defined using glob pattern. Works only for local files.
 
     Parameters
     ----------
-    filepath_or_buffer:
-        The filepath of the csv file.
-        We only support local files for now.
-    kwargs: Keyword arguments in pandas.read_csv
+    **kwargs : dict
+        Keyword arguments in `modin.pandas.read_csv`.
+
+    Returns
+    -------
+    Modin DataFrame.
```

Comment:
```suggestion
    modin.DataFrame
```
this should be a type, not description